src/features/script-load.js

```diff
@@ -84,5 +84,11 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
       }
     });
     document.head.appendChild(script);
+
+    // Note: when you use a script without `src`, the onload is not triggered after
+    // the script was added to the DOM.
+    if (!script.src) {
```

Comment:
This is a separate feature from async createScript hooks. I'm not against it, but it should be reviewed separately as it's completely unrelated to the rest of this PR. Could you move it to a separate PR? The main thing I want to check is whether new Event() works in IE11 or not.

Reply:
I remove it, I will create a new PR for this thanks.